src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/IContentRepository.kt

```diff
@@ -1,6 +1,7 @@
 package org.wycliffeassociates.otter.common.persistence.repositories
 
 import io.reactivex.Completable
+import io.reactivex.Maybe
```

Comment:
I'm guessing you don't need this import here

Reply:
good catch lol